gem/static/css/_enhanced.article-preview.scss

```diff
@@ -36,16 +36,42 @@
   }
 }
 
-.article-list {
+.article-list .article-preview  {
+  img {
+    width: 100%;
+  }
+  h3 {
+    margin: 0 0 9px 0;
+  }
+  p {
+    margin: 0;
+  }
+}
+
+.flexbox .article-list  {
   display: flex;
   flex-wrap: wrap;
+  .article-preview  {
+    padding: 0;
+    margin: 0 0 40px 0;
+    width: 50%;
+  }
+}
 
-
+.no-flexbox  .article-list  {
+  box-sizing: border-box;
```

Comment:
I don't think the device will support box-sizing if it doesn't support flex-box.